runtime/kusama/src/lib.rs

```diff
@@ -1484,7 +1507,7 @@ pub mod migrations {
 	);
 
 	/// Unreleased migrations. Add new ones here:
-	pub type Unreleased = ();
+	pub type Unreleased = (init_state_migration::InitMigrate,);
 }
```

Comment:
Your comment:
> It may be pertinent to avoid adding this in a runtime upgrade that already have others migration.

would be suited here so whoever does the move from `Unreleased` to `VNext` is aware of it.
We can then drop the comment.